workflow-bot-app/src/main/java/com/symphony/bdk/workflow/monitoring/repository/WorkflowInstQueryRepository.java

```diff
@@ -1,9 +1,10 @@
 package com.symphony.bdk.workflow.monitoring.repository;
 
+import com.symphony.bdk.workflow.api.v1.dto.StatusEnum;
 import com.symphony.bdk.workflow.monitoring.repository.domain.WorkflowInstanceDomain;
 
 import java.util.List;
 
 public interface WorkflowInstQueryRepository extends QueryRepository<WorkflowInstanceDomain, String> {
-  List<WorkflowInstanceDomain> findAllById(String id);
+  List<WorkflowInstanceDomain> findAllById(String id, StatusEnum status);
```

Comment:
maybe we can have two methods, one is findAllById, the other is findAllByIdAndStatus, it would be clearer and avoid passing null value for status arg